ADL-RVP/rt711-rt1316-rt714.asl

```diff
@@ -0,0 +1,45 @@
+ /** @file
```

Comment:
would you mind renaming the file as adl_sdca_3_in_1 to match what the Linux PR does: 

https://github.com/thesofproject/linux/pull/2811/

Reply:
@plbossart The patch is updated to rename to adl_sdca_3_in_1.asl